insights/mappers/xfs_info.py

```diff
@@ -1,16 +1,72 @@
+"""
+XFSInfo - command ``/usr/sbin/xfs_info``
+========================================
+
+This mapper reads the output of the ``xfs_info`` command.
+
+As this spec can collect more than one file, the mapper will return a list of
+XFSInfo objects, which need to be iterated through to find the information
+on the mount point or device you need.
+
+Attributes:
+    xfs_info(dict): A dictionary of dictionaries containing the data from the
+        report, keyed on the five section names in the output:
+        '``meta-data``', '``data``', '``naming``', '``log``', and
+        '``realtime``'.  '``meta-data``', '``data``' and '``log``' are always
+        present.  Within each dictionary a special key '``specifier``' stores
+        any data immediately after the section name - e.g. '/dev/sda' or
+        'version' in the case of the output below.  Any data immediately
+        following that is stored in the ``specifier_value`` key.  Otherwise,
+        data is read in key=value pairs - e.g. from the output below, the
+        ``isize`` key will have the value ``32`` (an integer).  Data values
+        given in blocks are left as is, so the value of the ``agsize`` key is
+        '16777184 blks' as a string.
+    mount(str): If the mount point can be derived from the file name of the
+        original output, then this attribute contains the reconstructed
+        mount point name.
+    device(str): The device name immediately after the '``meta-data``' section
+        heading.
+    data_size(int): The size of the data segment in bytes, from multiplying
+        the ``blocks`` and ``bsize`` values of the ``data`` section together.
+    log_size(int): The size of the log segment in bytes, from multiplying
+        the ``blocks`` and ``bsize`` values of the ``log`` section together.
```

Comment:
This attributes docstring should be moved to the `XFSInfo` class, which needs to have a docstring to be included in the documentation.  Having it in the module causes them to appear to be module attributes instead of class attributes.

There's also a warning generated from the doc build that needs to be fixed:
```
insights-core/insights/mappers/xfs_info.py:docstring of insights.mappers.xfs_info:46: WARNING: Unknown target name: "xfs_info".
```